src/eval-frame/components/reps/data-table-rep.jsx

```diff
@@ -64,7 +64,7 @@ class CellRenderer extends React.Component {
             : "none"
         }}
       >
-        <TinyRep serializedObj={serializeForTinyRep(this.props.value)} />
+        <TinyRep {...serializeForTinyRep(this.props.value)} />
```

Comment:
spread props out individually rather than passing down a big object with children. facilitates shallow comparison of props